app/services/base_index_job.rb

```diff
@@ -3,24 +3,46 @@ class BaseIndexJob
               :book_version_id,
```

Comment:
Did we switch to `index_strategy` being a string and `index_strategy_instance` being the object?  If so, should we go through and get all the `index_strategy_name`s out?

Reply:
@jpslav i thought we settled on the more semantic "index_strategy_name" (representing "I1").  just like search_strategy_name would be "S1".   and index_strategy and search_strategy would be the objects.